packages/trpc/server/routers/viewer/organizations/setPassword.handler.ts

```diff
@@ -0,0 +1,57 @@
+import { createHash } from "crypto";
+
+import { hashPassword } from "@calcom/features/auth/lib/hashPassword";
+import { verifyPassword } from "@calcom/features/auth/lib/verifyPassword";
+import { prisma } from "@calcom/prisma";
+
+import { TRPCError } from "@trpc/server";
+
+import type { TrpcSessionUser } from "../../../trpc";
+import type { TSetPasswordSchema } from "./setPassword.schema";
+
+type UpdateOptions = {
+  ctx: {
+    user: NonNullable<TrpcSessionUser>;
+  };
+  input: TSetPasswordSchema;
+};
+
+export const setPasswordHandler = async ({ ctx, input }: UpdateOptions) => {
```

Comment:
It being a user password change only, I think we should be able to reuse auth/changePassword